parakh-new/include/dbmodule.php

```diff
@@ -69,37 +69,41 @@ function getUserByEmail($email) {
     function getUserByLead($lead_id) {
         if (!filter_var($lead_id, FILTER_VALIDATE_INT) === false) {
             $employeeList = array();
-            $query = 'SELECT uh.user_id,u.google_name,u.google_email,u.mobile_number,u.designation,u.google_picture_link,u.google_email FROM user_hierarchy uh left join users u on u.id = uh.user_id ' .
+            $query = 'SELECT uh.user_id,u.google_name,u.google_email,u.mobile_number,u.designation,u.google_picture_link as picture,u.google_email FROM user_hierarchy uh left join users u on u.id = uh.user_id ' .
                     'WHERE manager_id = :id  AND u.status <> 0 group by user_id';
             $user_data = $this->con->prepare($query);
             $user_data->execute(array(':id' => $lead_id));
-            while ($row = $user_data->fetch((PDO::FETCH_ASSOC))) {
-                $membresInfo = array();
-                $query_rank = "SELECT MAX(r.created_date) as date,
+            $employeeList = $user_data->fetchAll((PDO::FETCH_ASSOC));
+            
+            $query_rank = "SELECT u.id,u.google_name,
                     sum(case when r.rating = 1 then 1  end) as pluscount,
```

Comment:
We can combine this in one single query:

SELECT uh.user_id,u.google_name,u.google_email,u.mobile_number,u.designation,u.google_picture_link as picture,u.google_email, 
sum(case when r.rating = 1 then 1  end) as pluscount,
                    sum(case when r.rating = 0 then 1  end) as minuscount
FROM user_hierarchy uh left join users u on u.id = uh.user_id left join rating r on (u.id=r.user_id)
WHERE manager_id = 66  AND u.status <> 0 group by r.user_id order by u.google_name